{"uuid": "3989021a-6acc-449f-83ab-1837d439b9c3", "children": ["4847026c-db08-43aa-80bb-799538208973"], "befores": [{"name": "get_key", "status": "failed", "statusDetails": {"message": "AssertionError: assert '2' == '1'\n  - 1\n  + 2\n", "trace": "  File \"C:\\Users\\syz\\AppData\\Local\\Programs\\Python\\Python37-32\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\syz\\AppData\\Local\\Programs\\Python\\Python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\syz\\AppData\\Local\\Programs\\Python\\Python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\python_Code\\pythonProject\\pythonclasses\\autoclass\\day10\\apiautotest\\conftest.py\", line 20, in get_key\n    return test_user_api.test_user_login()\n  File \"D:\\python_Code\\pythonProject\\pythonclasses\\autoclass\\day10\\apiautotest\\testcase\\test_user_api.py\", line 28, in test_user_login\n    assert res.json()[\"code\"] == \"1\"\n"}, "start": 1689262767470, "stop": 1689262767536}], "start": 1689262767470, "stop": 1689262767543}